lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -223,16 +238,32 @@ def import_kymotrackgroup_from_csv(filename, kymo, channel, delimiter=";"):
 
     Raises
     ------
-    IOError
+    ValueError
         If the file format is not as expected.
+
+    Examples
+    --------
+    ::
+
+        import lumicks.pylake as lk
+
+        file = lk.File("test_data/kymo.h5")
+        kymo = file.kymos["16"]  # Extract the kymo named 16
+
+        tracks = lk.track_greedy(kymo, channel="red", pixel_threshold=5)
+        tracks.save("tracks.csv")
+
+        loaded_tracks = lk.load_tracks("tracks.csv", kymo, "red")
```

Comment:
Maybe nice to add, as this would be the first thing I'd do when trying this function for the first time:

```
# Plot kymo and overlay imported tracks for comparison:
plt.figure()
kymo1.crop_by_distance(4.9,13.8).plot()
tracks.plot()
```

Reply:
Yeah, that's a good point. Considering we're tracking red, what do you think about plotting the red channel with the red tracks?

Something like this as example then:
```python
        import lumicks.pylake as lk

        file = lk.File("test_data/kymo.h5")
        kymo = file.kymos["16"]  # Extract the kymo named 16

        kymo_cropped = kymo.crop_by_distance(10, 25)
        tracks = lk.track_greedy(kymo_cropped, channel="red", pixel_threshold=5)
        tracks.save("tracks.csv")

        loaded_tracks = lk.load_tracks("tracks.csv", kymo_cropped, "red")

        # Plot the red channel of the kymograph and tracks together
        kymo_cropped.plot("red", adjustment=lk.ColorAdjustment(5, 95, "percentile"))
        loaded_tracks.plot()
```